Source/FakeItEasy/Core/DefaultArgumentConstraintManager.cs

```diff
@@ -23,6 +23,12 @@ public T Matches(Func<T, bool> predicate, Action<IOutputWriter> descriptionWrite
             return default(T);
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]

```

Comment:
Putting the attribute in this class will have no effect, because it's internal. Putting it in the interface should be enough.